### PR TITLE
[DOC release] Restore documentation for component helper

### PR DESCRIPTION
**Pull request by Gaurav0**
For https://github.com/emberjs/ember.js/issues/16638

### CODE REVIEW

File: packages/ember-glimmer/lib/helpers/component.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
+/**
+  @module ember
+*/
+
+/**
+  The `{{component}}` helper lets you add instances of `Component` to a
+  template. See [Component](/api/classes/Ember.Component.html) for
```
Comment (sivakumar-kailasam) [inline packages/ember-glimmer/lib/helpers/component.ts]: can we modify this link to link it to the new docs url?

Comment (Gaurav0) [reply packages/ember-glimmer/lib/helpers/component.ts]: OK. What is the new url?

Comment (sivakumar-kailasam) [reply packages/ember-glimmer/lib/helpers/component.ts]: https://emberjs.com/api/ember/release/classes/Component

Comment (Gaurav0) [reply packages/ember-glimmer/lib/helpers/component.ts]: thanks!

**Comment by sivakumar-kailasam** [comment]
LGTM 👍

**Comment by sivakumar-kailasam** [comment]
@locks please merge :)

**Comment by toddjordan** [comment]
@sivakumar-kailasam you think we should pull this branch and see if docs can get generated right for it?  @Gaurav0 have you tried locally?  I realize local setup for api docs testing is a bear.  I could maybe try this evening.

**Comment by Gaurav0** [comment]
@toddjordan No. Is running the backend necessary? Is there a document listing all the steps?

**Comment by sivakumar-kailasam** [comment]
@toddjordan If you have the time sure :) But this looks similar to https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/lib/helpers/loc.ts so i'm guessing it should be fine. 

But curious to know where this would come up in the app. with search broken atm, its hard to figure out where it'd be

**Comment by toddjordan** [comment]
https://github.com/ember-learn/ember-jsonapi-docs/blob/master/README.md#generating-api-documentation-and-testing-api-docs-locally

**Comment by Gaurav0** [comment]
@sivakumar-kailasam Is there a way in yuidoc to specify a package? 

Maybe it would it be put in ember-glimmer because it is in that directory?

@toddjordan Thank you.

**Comment by Gaurav0** [comment]
Note: The documentation previously appeared here: https://www.emberjs.com/api/ember/3.0/classes/Ember.Templates.helpers/methods/component?anchor=component

**Comment by Gaurav0** [comment]
Update: I expect it to appear on this page: https://www.emberjs.com/api/ember/3.2/classes/Ember.Templates.helpers

but I don't know how to get there by clicking from https://www.emberjs.com/api

**Comment by Gaurav0** [comment]
@sivakumar-kailasam @toddjordan @locks 

Never mind! Ran docs locally. It works!

screenshot from 2018-06-21 15-47-51

screenshot from 2018-06-21 15-48-21

**Comment by toddjordan** [comment]
excellent! thanks @Gaurav0

**Comment by jherdman** [comment]
Hey gang. This seems to have been lost in the 3.2 docs:

screen shot 2018-07-04 at 3 33 22 pm

**Pull request merged**